app/services/nomis/booking.rb

```diff
@@ -0,0 +1,17 @@
+module Nomis
+  class Booking
+    include NonPersistedModel
+
+    attribute :visit_id, Integer
+    attribute :error_message, String
+
+    def self.build(response)
+      if response.key?('error')
+
+        new(error_message: response['error']['message'])
```

Comment:
`ActiveModel::Error` is for validations so not a fan of adding outside the validation lifecycle. We use `.build` somewhere else to translate the api call response